src/ape/managers/accounts.py

```diff
@@ -317,3 +336,12 @@ def __contains__(self, address: AddressType) -> bool:
             any(address in container for container in self.containers.values())
             or address in self.test_accounts
         )
+
+    def use_sender(self, account, passphrase: Optional[str] = None):
+        if not isinstance(account, AccountAPI):
+            try:
+                account = self.load(account)  # if account is alias
+            except (ValueError, IndexError):
```

Comment:
Instead of `try` / `except`, check if it is a hex via the `is_hex` method from `eth_utils`

Reply:
in this scenario account is alias so just string, i would do that if account was just address, on the other hand, account could be even int